docs/api/session.md

```diff
@@ -1091,6 +1091,55 @@ app.whenReady().then(() => {
 })
 ```
 
+#### `ses.setUSBProtectedClassesHandler(handler)`
+
+* `handler` Function\<string[]> | null
+  * `details` Object
+    * `protectedClasses` string[] - The current list of protected USB classes. Possible class values are:
+      * `audio`
+      * `audio_video`
```

Comment:
Maybe use `audio-video` instead, our APIs have mixed styles for names though so I'm not sure which style should we stick to.

Reply:
I think you are right.  If we follow https://w3ctag.github.io/design-principles/#casing-rules, dashes should be used.